addons/message_paid_research/manifest.json

```diff
@@ -31,11 +31,11 @@
         "content": [
           {
             "id": "l_1",
-            "content": "If you are invited to the study, you will be asked to log in to our research platform and complete 2-3 short activities. We estimate that it will take about 30 minutes of your time to complete them all, and you would be compensated $60."
+            "content": "If you are invited to the study, you will be asked to log into our research platform and complete 4 short activities over the course of 2 days, such as reviewing a new prototype and participating in discussions about new ideas."
```

Comment:
Passing by note on "log into"
https://acorn.firefox.com/latest/content/word-list/g-m.html#section-log-in-login-e0